test/LoggingTest.js

```diff
@@ -0,0 +1,75 @@
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var winston = require('winston');
+var Config = require('../lib/Config');
+var HazelcastClient = require('../lib/HazelcastClient');
+describe('Logging Test', function() {
+    var client;
+
+    var winstonAdapter = {
+        logger: new (winston.Logger)({
+            transports: [
+                new (winston.transports.Console)()
+            ]
+        }),
+
+        levels: [
+            'error',
+            'warn',
+            'info',
+            'debug',
+            'silly'
```

Comment:
those are winston logging library's levels. purpose of this code piece is to map our levels to winston's. Our levels are 
error
warn
info
debug
trace